dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/PolicyExecutorImpl.java

```diff
@@ -1499,8 +1500,9 @@ public void updateConfig(Map<String, Object> props) {
         boolean useVirtualThreads = null == (v = props.get("virtual")) ? false : (Boolean) v;;
 
         // Validation that cannot be performed by metatype:
-        if (useVirtualThreads && virtualThreadOps == null)
-            throw new IllegalArgumentException("virtual: true");
+        if (useVirtualThreads && !virtualThreadOps.isSupported()) {
+            useVirtualThreads = false;
+        }
```

Comment:
Restore code to reject an invalid value,
```suggestion
        if (useVirtualThreads && !virtualThreadOps.isSupported()) {
            throw new IllegalArgumentException("virtual: true");
        }
```

Reply:
Did this get resolved accidentally? It looks like the code is still there.